<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wnd_main</class>
 <widget class="QMainWindow" name="wnd_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>e-draw</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/res.qrc">
    <normaloff>:/icons/icons/logotype.png</normaloff>:/icons/icons/logotype.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="vlt_top"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mbr_tools">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="men_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="act_saveas">
     <property name="title">
      <string>Save As...</string>
     </property>
    </widget>
    <addaction name="act_saveas"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="men_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_help"/>
    <addaction name="act_about"/>
   </widget>
   <addaction name="men_file"/>
   <addaction name="men_help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string notr="true">toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_color"/>
   <addaction name="act_width"/>
   <addaction name="separator"/>
   <addaction name="act_add"/>
   <addaction name="act_undo"/>
   <addaction name="separator"/>
   <addaction name="act_clear"/>
  </widget>
  <action name="act_color">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/icons/icons/color_act.png</normaloff>:/icons/icons/color_act.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="toolTip">
    <string>Change pen color   Shift+C</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="act_width">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/icons/icons/width_act.png</normaloff>:/icons/icons/width_act.png</iconset>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="toolTip">
    <string>Change pen width   Shift+W</string>
   </property>
   <property name="shortcut">
    <string>Shift+W</string>
   </property>
  </action>
  <action name="act_add">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/icons/icons/add_act.png</normaloff>:/icons/icons/add_act.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add current image to session   Shift+A</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="act_undo">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/icons/icons/undo_act.png</normaloff>:/icons/icons/undo_act.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo latest stroke   Ctrl+Z</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="act_clear">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/icons/icons/clear_act.png</normaloff>:/icons/icons/clear_act.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear the canvas   Shift+X</string>
   </property>
   <property name="shortcut">
    <string>Shift+X</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="act_help">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_exit</sender>
   <signal>triggered()</signal>
   <receiver>wnd_main</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
